Topics/Tech_Stacks/Tailwind.md

```diff
@@ -0,0 +1,101 @@
+# Getting Stylish and Responsive with Tailwind CSS
+
+## Table of Contents
+### [Introduction](#introduction)
+### [Installation](#installation)
+### [Usage](#usage)
+### [Resources](#resources)
+### [Flexibility](#flexibility)
+### [Limitations](#limitations)
+
+## Introduction
+Tired of the nuances of CSS? Tailwind is a utility-first CSS framework, which allows developers to use premade styling classes, without the need to write CSS classes from scratch. Tailwind provides small, single-purpose, reusable classes for spacing, lettering, and colours. These classes can then be used in HTML elements, or components from frontend libraries and frameworks.
+
+## Installation
+1. When using node, install and execute Tailwind:
+```bash
+npm install -D tailwindcss
+npx tailwindcss init
+```
+
+2. The generated `tailwind.config.js` configuration file contains the default setup. Give Tailwind access to files by listing the template paths in the content section. For React/TypeScript web app, it can look like this:
+```js
+content: [
+    './pages/**/*.{ts,tsx}',
+    './components/**/*.{ts,tsx}',
+    './app/**/*.{ts,tsx}',
+    './src/**/*.{ts,tsx}',
+  ],
+```
+
+3. Add the following to the global CSS file:
+```css
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+```
+
+Now we're set!
+
+## Usage
+
+### The Basics
+Each Tailwind class is focused on a specific responsibility. To use a class, add it to the `class` or `className` (when using React) attribute of the element/component. For example, use the following Tailwind classes to create a div that
+- occupies the full width of the screen
```

Comment:
```suggestion
- occupies the full width of the screen (ie. `w-screen`)
```

I'd recommend adding the example classes that you showcase below so that they can clearly draw the connection.

Reply:
I think that improves clarity. Added!